### PR TITLE
Added LinkPager widget and fixed Progress rendering

### PR DESCRIPTION
**Pull request by simialbi**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️ 
| New feature?  | ✔️ 
| Breaks BC?    | ❌ 
| Tests pass?   | ✔️ 
| Fixed issues  | #120, #121 

Progress bar width percentage does not getting rendered with formatter any more
Demo: http://demo.karlen.li/web/site/test

### CODE REVIEW

File: src/LinkPager.php [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+
+        if ($active) {
+            Html::addCssClass($options, $this->activePageCssClass);
+        }
+        if ($disabled) {
+            Html::addCssClass($options, $this->disabledPageCssClass);
+            $disabledItemOptions = $this->disabledListItemSubTagOptions;
+            $linkOptions = ArrayHelper::merge($linkOptions, $disabledItemOptions);
+            $linkOptions['tabindex'] = '-1';
+//            $tag = ArrayHelper::remove($disabledItemOptions, 'tag', 'span');
```
Comment (samdark) [inline src/LinkPager.php]: Remove?

File: src/LinkPager.php [hunk 2]
```diff
@@ -0,0 +1,304 @@
+<?php
+/**
+ * @package yii2-bootstrap4
+ * @author Simon Karlen <simi.albi@gmail.com>
+ */
```
Comment (rob006) [inline src/LinkPager.php]: This license header is incorrect. it should be:

```php
/**
 * @link http://www.yiiframework.com/
 * @copyright Copyright (c) 2008 Yii Software LLC
 * @license http://www.yiiframework.com/license/
 */
```

File: tests/LinkPagerTest.php [hunk 3]
```diff
@@ -0,0 +1,154 @@
+<?php
+/**
+ * @package yii2-bootstrap4
+ * @author Simon Karlen <simi.albi@gmail.com>
+ */
```
Comment (rob006) [inline tests/LinkPagerTest.php]: This license header is incorrect. it should be:

```php
/**
 * @link http://www.yiiframework.com/
 * @copyright Copyright (c) 2008 Yii Software LLC
 * @license http://www.yiiframework.com/license/
 */
```

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**